<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>469</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Программа управления сканером</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="speedSlider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1050</number>
    </property>
    <property name="singleStep">
     <number>3</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="sliderPosition">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
   </widget>
   <widget class="QLabel" name="joystickXYLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>(x, y)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="speedLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Скорость</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveUpButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>▲</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveDownButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>▼</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveLeftButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>◀</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveRightButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>▶</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>400</y>
      <width>371</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string>%p%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveToButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Переместить</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentYLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentXLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="targetXEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="targetYEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pointWidthEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pointHeightEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scanButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>360</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Сканировать</string>
    </property>
   </widget>
   <widget class="QLabel" name="setHeightLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Высота</string>
    </property>
   </widget>
   <widget class="QLabel" name="setWidthLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Ширина</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="resolutionEdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="setResolutionLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Разрешение</string>
    </property>
   </widget>
   <widget class="QLabel" name="progressLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Прогресс</string>
    </property>
   </widget>
   <widget class="QLabel" name="pointsLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>точек</string>
    </property>
   </widget>
   <widget class="QLabel" name="stepsLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>150</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>шагов</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setToZeroButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Обнулить координаты</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="enableXCheckBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Включить двигатель X</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="enableYCheckBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Включить двигатель Y</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="stepWidthEdit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>130</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="stepHeightEdit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>160</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="zigzagRadioButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Зигзаг</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="spiralRadioButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>220</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Спираль</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="integrationTimeEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>50000</string>
    </property>
   </widget>
   <widget class="QLabel" name="integrationTimeLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Время накопления (мкс)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="virtualSpecCheckBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>340</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Виртуальный спектрометр</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="scanToAverageEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>310</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="scanToAverageLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>290</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Число измерений для усреднения</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>452</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
